src/BaGet/BaGet.csproj

```diff
@@ -18,6 +18,7 @@
     <PackageReference Include="Microsoft.EntityFrameworkCore.Sqlite" Version="2.2.0" />
     <PackageReference Include="Microsoft.Extensions.Logging.Configuration" Version="2.2.0" />
     <PackageReference Include="Microsoft.Extensions.Logging.Console" Version="2.2.0" />
+    <PackageReference Include="Npgsql.EntityFrameworkCore.PostgreSQL" Version="2.1.2" />
```

Comment:
Should this be bumped up to version 2.2.0? https://www.nuget.org/packages/Npgsql.EntityFrameworkCore.PostgreSQL/2.2.0